gh2gcs: add initial documentation (#1338)
kubernetes/release: cmd/gh2gcs/README.md

# GitHub Releases to Google Cloud Storage

This directory contains a tool called `gh2gcs` that downloads a release published in GitHub to a Google Cloud Storage Bucket.

## Requirements

> This have a `gcloud` (`gsutil`) dependency. You need to install, if you not have that installed in your system.

Reviewer: Nit: 
```suggestion
This tool directly depends on `gcloud` and `gsutil` to be installed on the system.
```

Author: done
